app/classes/textile.rb

```diff
@@ -362,6 +383,23 @@ def pre_existing_links_replaced_by_placeholders!(do_object_links)
     saved_links
   end
 
+  IMPLICIT_TERM_PATTERN = /
+    (?<prefix> ^|\W) # prefix
+    (?:_+)
+    (?<id> [\p{Alpha}\-.\ ]+) # alpha chrs, hyphens, periods, spaces
```

Comment:
Do we care about diacritics?  Does the `{Alpha}` class already handle them?

Reply:
1. \p{Alpha} does handle diacritics.
```ruby
irb(main):002:0> /\p{Alpha}+/ =~ "olé"
=> 0
irb(main):003:0> $LAST_MATCH_INFO
=> #<MatchData "olé">
```
2. Do we care about them? 
It's easy enough to handle them. And I can think of at least potential terms than include them.
Ex: Köhler illumination